Allow ability to disable Redis capturing the args (#1276)
DataDog/dd-trace-rb: docs/GettingStarted.md

| --- | ----------- | ------- |
| `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
| `service_name` | Service name used for `redis` instrumentation | `'redis'` |

> | `show_command_args` | Show the command args (e.g. `GET key`) as a resource & tag | true |

Reviewer: Nit documentation suggestion:
```suggestion
| `command_args` | Show the command arguments (e.g. `key` in `GET key`) as resource name and tag | true |
```

Author: Renamed to `command_args` @marcotc
